docs/tutorials/vertex_nomination/SpectralVertexNomination.ipynb

```diff
@@ -0,0 +1,308 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "nomination_matrix, distance_matrix = svn.predict()\n",
+    "color = ['red', 'lightgreen', 'gold', 'cyan', 'pink']\n",
+    "seed_color = ['firebrick', 'green', 'tan', 'darkblue', 'purple']\n",
+    "plt.figure(figsize=(12, 9))\n",
+    "plt.scatter(svn.embedding_[seed, 0], svn.embedding_[seed, 1], c='black')\n",
+    "for i in range(nomination_matrix.shape[1]):\n",
+    "    plt.scatter(svn.embedding_[seed[i], 0],\n",
+    "                svn.embedding_[seed[i], 1],\n",
+    "                c=seed_color[i])\n",
+    "    plt.scatter(svn.embedding_[nomination_matrix[1:6, i], 0],\n",
+    "                svn.embedding_[nomination_matrix[1:6, i], 1],\n",
+    "                c=color[i])\n",
+    "    plt.xticks([], [])\n",
+    "    plt.yticks([], [])\n",
+    "    plt.title('Top 5 Nominees For Each Seed (black) in First Two Embeded Dimensions')"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "In the attributed case, seed vertices are labelled with an attribute. Here this is achieved by passing the seed as a two dimensional array of seed indices and attributes.\n",
+    "Now, instead of nominating for each seed vertex, we want to nominate for each unique attribute present in the seed set.\n",
+    "\n",
+    "Therefore, in the attributed case, our nomination list will be a $|V|$ x $m$ matrix where $m$ is the number of unique attributes in the seed $S$.\n",
+    "In order to achieve this, we need a way to quantity the distance from $v \\in V$ to $A_i \\subset S, 0 \\leq i < m$.\n",
+    "\n",
+    "A simple way of doing this is by summing the inverses of the distance from $v$ to each vertex in the attribute's set.\n",
```

Comment:
Just averaging doesn't work, because it loses all information about the number of neighbors belonging to each attribute. E.g. if $v$ has one neighbor of with attribute 0 at distance 1 and three neighbors of attribute 2 at distance 1.25, averaging will say  attribute 0 is closest ($mean(1) < mean(1.25, 1.25, 1.25)$ but the metric I used will say attribute 1 is closest as expected ($(1^{-1})^{-1) = 1   >  (1.25^{-1} + 1.25^{-1} + 1.25^{-1})^{-1} = .417)